Templates/Common-Backend-Scripts/wazideploy-generate.sh

```diff
@@ -339,16 +451,11 @@ validateOptions() {
     echo $ERRMSG
   else
     # check for relative path
-    if [[ ! ${PackageInputFile:0:1} == "/" ]] ; then 
-        checkWorkspace
-        PackageInputFile="$(getLogDir)/${PackageInputFile}"
+    if [ ! ${PackageInputFile:0:1} == "/" ] && [ -z "${usePackageUrl}" ]; then
```

Comment:
ArchiveInputFile? I know WD uses packageInputFile... 
No need to change...

Reply:
With introducing the Archive - we already start with a new terminology. We should apply the Wazi Deploy terms here.